doc/router_certificates.md

```diff
@@ -0,0 +1,31 @@
+# Router certificates
+
+The default KubeCF installation generates a `router_ca` and a `router_ssl` variables. The former
+is the CA certificate that signs the later, which in turn is used in the gorouter for TLS
+termination.
+
+For production systems, where certificates signed by a well-known Certificate Authority is required,
+KubeCF allows those certificates to be passed via Helm properties. To do so, set the values:
```

Comment:
Hmm, wouldn't _production_ systems want either ingress or external SSL termination? I guess some shops want as much traffic as possible encrypted, so they punt the decryption to the router…

Reply:
I don't think so. In some setups, it would make sense to have the ingress-controller to take care of a broader set of endpoints. For that, we already have some docs and is out of scope for this PR. For other SSL termination, especially the ones terminated at the public cloud load balancers, I consider them extraneous and believe them to be out of scope for this document and especially this PR.